src/button/template/componentTemplate.jsx

```diff
@@ -120,14 +124,26 @@ function renderContent(text : string, { label, locale, color, branding, logoColo
                 return;
             }
 
-            if (!logoColor) {
+            if (!logoColor && !isFundingThrottleEnabled) {
                 throw new Error(`Can not determine logo without logo color`);
             }
+
+            if (isFundingThrottleEnabled && layout !== 'vertical') {
+                if (label === 'credit' && (name === 'paypal' || name === 'pp')) {
+                    return;
+                }
+
+                if (name === 'credit') {
```

Comment:
Same here -- would be good to use constants.

Reply:
Done.